apps/emqx_rule_engine/test/emqx_rule_monitor_SUITE.erl

```diff
@@ -0,0 +1,95 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2020 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-module(emqx_rule_monitor_SUITE).
+
+-compile(export_all).
+-compile(nowarn_export_all).
+
+-include_lib("emqx_rule_engine/include/rule_engine.hrl").
+-include_lib("emqx/include/emqx.hrl").
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("common_test/include/ct.hrl").
+
+all() ->
+    [ {group, resource}
+    ].
+
+suite() ->
+    [{ct_hooks, [cth_surefire]}, {timetrap, {seconds, 30}}].
+
+groups() ->
+    [{resource, [sequence],
+      [ t_restart_resource
+      ]}
+    ].
+
+init_per_suite(Config) ->
+    ok = ekka_mnesia:start(),
+    ok = emqx_rule_registry:mnesia(boot),
+    Config.
+
+end_per_suite(_Config) ->
+    ok.
+
+t_restart_resource(_) ->
+    {ok, _} = emqx_rule_monitor:start_link(),
+    Opts = [public, named_table, set, {read_concurrency, true}],
+    _ = ets:new(?RES_PARAMS_TAB, [{keypos, #resource_params.id}|Opts]),
+    ets:new(t_restart_resource, [named_table, public]),
```

Comment:
create ets when case `init` step, and `ets:delete` in `'end'` step.

Reply:
OK, I will change this.